test/e2e/flask/multichain-api/wallet_invokeMethod.spec.ts

```diff
@@ -251,4 +254,258 @@ describe('Multichain API', function () {
       });
     });
   });
+
+  describe('EIP-5792 Methods', function () {
+    describe('Calling `wallet_getCapabalities`', function () {
+      it('should return the available capabilities', async function () {
+        await withFixtures(
+          {
+            ...DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+            title: this.test?.fullTitle(),
+            fixtures: new FixtureBuilder()
+              .withPermissionControllerConnectedToMultichainTestDapp()
+              .build(),
+            localNodeOptions: [
+              {
+                type: 'anvil',
+                options: {
+                  hardfork: 'prague',
+                  loadState:
+                    './test/e2e/seeder/network-states/eip7702-state/withDelegatorContracts.json',
+                },
+              },
+            ],
+            testSpecificMock: mockEip7702FeatureFlag,
+          },
+          async ({ driver, extensionId }: FixtureCallbackArgs) => {
+            const scope = GANACHE_SCOPES[0];
+            const method = 'wallet_getCapabilities';
+
+            await unlockWallet(driver);
+
+            const testDapp = new TestDappMultichain(driver);
+            await testDapp.openTestDappPage();
+            await testDapp.connectExternallyConnectable(extensionId);
+            await testDapp.initCreateSessionScopes([scope]);
```

Comment:
We should probably add to the fixtures so we don't have to go through the steps of initiating the session each time since its not necessary for testing what we want to target, but that is the pattern we have in these e2e test so far so we can separate out that as a refactor